### ISSUE TITLE
Add authors to AUTHORS file

### ISSUE THREAD

**Issue by drjwbaker** [data-lessons/library-openrefine-DEPRECATED#131]
We now have a workflow for releasing citable versions of our lessons (with DOIs) every 6 months via Zenodo. This makes our more discoverable and sustainable and ensures that everyone involved gets the credit they deserve. For more on this work see https://github.com/data-lessons/librarycarpentry/issues/5

In order to make this happen we need to make one crucial change: all AUTHORS files need to change so that they list names of contributors in the following format:

`James Allen`
`James Baker`
`Piotr Banaszkiewicz`
`Erin Becker`

@jt14den will run a script that that strips names from lesson logs and edit AUTHORS across all Library Carpentry repos.

When this is actioned (hopefully, soon!), lesson maintainers are asked to eyeball the AUTHORS file to see if anyone obvious is missing (for example, people who contributed to discussions but didn't edit any lessons). Note: template developers are credited in this process; this is in line with Software Carpentry best practice.

In the future, lesson maintainers are encouraged to ensure that those who contribute to lessons are added manually to AUTHORS files (encourage contributors to do it so they see where and how we give credit!)

**Comment by ccronje** [comment]
@jt14den can you please let us know if this has been actioned?

**Comment by jt14den** [comment]
@ccronje I moved the publish date to July - the script I'm using sets the release date as YYYY-MM -- and I figured we need a little review time for maintainers. What James wrote above has changed a bit: the release script will create a branch named 2017-07 and push up to the repo. You'll need to review the AUTHORS (the script will populate that file) in that branch and I'll pull down changes prior to publishing. I should have time to work on this this week. I'll ping maintainers on the separate repos when we are ready (wish there was an "at" maintainers).
